regression/memory-analyzer/structs_04/test.desc

```diff
@@ -0,0 +1,6 @@
+KNOWNBUG
```

Comment:
As above.

Reply:
Explained (single char extraction unsupported for now).